uint16 id
bool[] bool_args
uint16[] uint_args
string[] string_args
---
